apps/builder/app/routes/dashboard._index.tsx

```diff
@@ -51,6 +56,15 @@ export const ErrorBoundary = () => {
 };
 
 const DashboardRoute = () => {
+  // Allows to set feature flags via URL parameter features=name1,name2,name3
```

Comment:
- why do we need to depend on react for this?
- why is this only in dashboard? what if URL is directly to a project?

I think we could solve both by simply checking this somewhere in feature-flags module

Reply:
you are right, adding aside effect to the module is bad, I guess then we have to move this to a hook and use it in both dashboard and builder

Reply:
ah I know what you mean now, but that's bad too, so lets just use what you did as a hook from feature-flags and call it in both dashboard and builder

Reply:
looking good, you are right, we need to merge them, otherwise it could be more surprises